test/circuit/library/test_raw_feature_vector.py

```diff
@@ -92,17 +92,10 @@ def test_partially_bound(self):
 
     @unittest.skipUnless(optionals.HAS_AER, "qiskit-aer is required to run this test")
     def test_usage_in_vqc(self):
-        """Test using the circuit the a single VQC iteration works."""
+        """Test the circuit works in VQC."""
```

Comment:
We no longer need the skipUnless AER decorator right,

Reply:
Agree, removed the decorator.